build.sc

```diff
@@ -1,23 +1,23 @@
-import $ivy.`de.tototec::de.tobiasroeser.mill.integrationtest::0.6.0`
+import $ivy.`de.tototec::de.tobiasroeser.mill.integrationtest::0.6.1`
 import de.tobiasroeser.mill.integrationtest._
-import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version::0.1.4`
+import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version::0.3.0`
 import de.tobiasroeser.mill.vcs.version.VcsVersion
 import mill._
 import mill.scalalib._
 import mill.scalalib.api.Util.scalaNativeBinaryVersion
 import publish._
 
-val millVersions                           = Seq("0.10.5", "0.9.12")
+val millVersions                           = Seq("0.10.8", "0.9.12")
 def millBinaryVersion(millVersion: String) = scalaNativeBinaryVersion(millVersion)
 
 object `mill-tpolecat` extends Cross[MillTpolecatCross](millVersions: _*)
 class MillTpolecatCross(millVersion: String) extends CrossModuleBase with PublishModule {
-  override def crossScalaVersion = "2.13.7"
+  override def crossScalaVersion = "2.13.10"
   override def artifactSuffix    = s"_mill${millBinaryVersion(millVersion)}" + super.artifactSuffix()
 
   def compileIvyDeps = Agg(ivy"com.lihaoyi::mill-scalalib:$millVersion")
 
-  def publishVersion: T[String] = VcsVersion.vcsState().format(tagModifier = _.dropWhile(_ == 'v'))
```

Comment:
So stripping the `v` is no longer necessary as this is done by default now. However, it got me wondering. I don't see any git tags associated with this repo. Are you just manually tagging via GitHub or something or how are you releasing?

Reply:
Hi @ckipp01 I've not been so involved with this repo recently but as far as I know we're just using git tags for releasing on tag pushes
![image](https://user-images.githubusercontent.com/2992938/211068885-82f7baae-332c-4ffc-a5b3-9f5ad2921f64.png)

Reply:
Sorry I missed this PR. I did not quiet understand the next steps. Is `publishVersion` defined by default without `v` now?

Reply:
> Hi @ckipp01 I've not been so involved with this repo recently but as far as I know we're just using git tags for releasing on tag pushes

Yup, this was my bad, I hadn't fetched everything upstream and was only dealing with my fork. Just ignore that part.

> Sorry I missed this PR. I did not quiet understand the next steps. Is publishVersion defined by default without v now?

Well the logic we had here was stripping the `v` which as of 0.2.0 is done by [default](https://github.com/lefou/mill-vcs-version/releases/tag/0.2.0).